src/OrchardCore.Modules/OrchardCore.Autoroute/Models/AutoroutePart.cs

```diff
@@ -4,6 +4,9 @@ namespace OrchardCore.Autoroute.Models
 {
     public class AutoroutePart : ContentPart
     {
+        public static char[] InvalidCharactersForPath = ":?#[]@!$&'()*+,.;=<>\\|%".ToCharArray();
+        public const int MaxPathLength = 1024;
```

Comment:
idem same

Reply:
@deanmarcussen thanx for the review. I moved field validation constants to Startup, although they shouldn't be part of the JSON. Maybe introducing a class like AutoroutePart.Fields would be a better option? Or in Migrations?

Reply:
> they shouldn't be part of the JSON

Maybe a `[JsonIgnore]` attribute

Reply:
Thanx for the tip @jtkech. I really prefer having the constants in Part model class instead of startup. They are ContentPart's characteristics and they are available on all places that use the part. I did not decorate them with [JsonIgnore] because it is only applicable on properties. Also, JSON.Net does not serialize constants/static members out of the box unless they are decorated with [JsonProperty] attribute. Could not find any custom converter for ContentPart in source that would force serialization of consts/static members. So I moved them back for now.